test/support/live_views/general.ex

```diff
@@ -5,6 +5,7 @@ defmodule Phoenix.LiveViewTest.ThermostatLive do
 
   def render(assigns) do
     ~L"""
+    Flash: <%= live_flash(@flash, :info) %>
```

Comment:
So the views in this test case already handle tons of concerns. My suggestions are to either:

1. Handle this in the params.ex views since they already handle a bunch of scenarios for redirect
2. Introduce a new series of fixtures for flash exclusively